test/testlist.txt

```diff
@@ -23,5 +23,7 @@ unit/services/azureservicebus/deprovision-spec.js
 unit/services/azureservicebus/bind-spec.js
 unit/services/azureservicebus/unbind-spec.js
 
+unit/services/azuresqldb/provision-spec.js
+
 #integration/catalog.js
 #integration/lifecycle.js
```

Comment:
Could you please update the test matrix for lifecycle test?

A PR will be submitted by me to automate the lifecycle test. And I've submitted a code review to you. Please have a look.